lua/entities/train_fucked/init.lua

```diff
@@ -0,0 +1,13 @@
+AddCSLuaFile( "cl_init.lua" ) -- Make sure clientside
+AddCSLuaFile( "shared.lua" )  -- and shared scripts are sent.
+include('shared.lua')
+ 
+function ENT:Initialize()
+
+	self:SetModel( "models/props_trainstation/train001.mdl" )
+	self:PhysicsInit( SOLID_VPHYSICS )
+	self:SetSolid( SOLID_VPHYSICS )
```

Comment:
```suggestion
    self:SetModel( "models/props_trainstation/train001.mdl" )
    self:PhysicsInit( SOLID_VPHYSICS )
    self:SetSolid( SOLID_VPHYSICS )
```
Looks like some tabs got in :o
Also no need for newline at start (or end, line 12) of function